src/providers/data_provider_be.c

```diff
@@ -588,7 +588,7 @@ errno_t be_process_init(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
-    be_ctx->sbus_name = sss_iface_domain_bus(be_ctx, be_ctx->domain);
+    be_ctx->sbus_name = confdb_get_domain_bus(be_ctx, be_ctx->domain);
```

Comment:
Here and everywhere: why not `be_ctx->domain->conn_name`?

Then `confdb_get_domain_bus()` could be made internal/static, and probably change of `libsss_util_la_SOURCES` wouldn't be needed.

Reply:
Nice catch. This was some hard refactoring but it is done. utils sources still must keep sbus_opath.c.